src/pythainer/examples/builders/__init__.py

```diff
@@ -241,6 +241,68 @@ def vulkan_builder() -> PartialDockerBuilder:
     return builder
 
 
+def vTune_builder() -> PartialDockerBuilder:
+    """
+    Configures a Docker builder so it can use the intel vtune profiler.
+    Due to how this implementation works you will need to do adaptations to the host system.
+    One such is:
+    Run "echo "0" | sudo tee /proc/sys/kernel/yama/ptrace_scope > /dev/null" on host
+        -> this comes with securety concerns only do if you know what you are dealing with
```

Comment:
could you reformulate and be a bit more precise? "security concerns" is a bit vague